### PR TITLE
added section title support above touchable element

### PR DESCRIPTION
**Pull request by jeffreymendez1993**
Added section title support.

### CODE REVIEW

**Comment by iRoachie** [comment]
Could you describe a use case where this would be useful?

**Comment by jeffreymendez1993** [comment]
Sure. If I want to dynamically create collapsible sections with section titles this is a nice feature.
screen shot 2018-02-13 at 8 23 03 am

**Comment by jeffreymendez1993** [comment]
No way to make sections have there own titles without it being in the touchable element. I could wrap it in a higher order but this is a clean solution that others I think can benefit from.

**Comment by iRoachie** [comment]
Alright, makes sense. Can you rebase this PR from the `next` branch

**Comment by jeffreymendez1993** [comment]
No problem will do now. @iRoachie

**Comment by jeffreymendez1993** [comment]
@iRoachie There we go.

File: README.md [hunk 1]
```diff
@@ -58,8 +60,7 @@ import Accordion from 'react-native-collapsible/Accordion';
 | **`disabled`**                                          | Set whether the user can interact with the Accordion                                                            |
 | **`align`**                                             | See `Collapsible`                                                                                               |
 | **`duration`**                                          | See `Collapsible`                                                                                               |
-| **`easing`**                                            | See `Collapsible`                                                                                               |
-
+| **`easing`**                                            | See `Collapsib
```
Comment (iRoachie) [inline README.md]: Typo here

Comment (jeffreymendez1993) [reply README.md]: fixed

**Comment by jeffreymendez1993** [comment]
when can this be merged?

**Pull request merged**